src/actions/traderAction.js

```diff
@@ -164,3 +172,184 @@ export async function convertToken(token) {
 
   return result[0][1];
 }
+
+export async function getBurnedNEC() {
+  const account = await web3.eth.getAccounts().then(res => res[0]);
+  const engineContract = new web3.eth.Contract(abis.engineContract, config.necEngineContract, {
+    from: account,
+  });
+
+  return engineContract.methods
+    .getNextAuction()
```

Comment:
I understand this call is not required here. this was added to test if web3 was initialising and making calls. This will now be updated to retrieve and process data.

Reply:
Great np. My only more general comment on all this is whether this should be inside traderAction (instead of having its own actions file)

Reply:
I assumed the auction functionality will be used by the traders. Will separate it out as auction i the next commit.